vendor/github.com/golang/glog/glog.go

```diff
@@ -676,10 +676,7 @@ func (l *loggingT) output(s severity, buf *buffer, file string, line int, alsoTo
 		}
 	}
 	data := buf.Bytes()
-	if !flag.Parsed() {
```

Comment:
This is a rather ugly fix. It would be better to fork glog if we want to keep it for a while. But I decided to keep a change of the logging lib out of this PR. Let's move on for now.